intro-HPC/ch_running_jobs_with_input.tex

```diff
@@ -207,21 +208,30 @@ \subsection{Your home directory (\$VSC\_HOME)\label{subsec:home-directory}}
 .bash\_history             & This file contains the commands you typed at your shell prompt, in case you need them again. \\ \hline
 \end{tabular}
 
+\ifgent
+% we don't auto create these files in gent
+\else
 Furthermore, we have initially created some files/directories there (tutorial,
 docs,  examples, examples.pbs) that accompany  this manual and allow you to
 easily execute the provided examples.
+\fi
 
 \subsection{Your data directory (\$VSC\_DATA)\label{subsec:data-directory}}
 
 In this directory you can store all other data that you need for longer terms
-(such as the results of previous jobs, \ldots). It is a good place for, e.g., 
-conducting large software builds.
+(such as the results of previous jobs, \ldots). It is a good place for, e.g.,
+storing big files like genome data.
 
-The environment variable pointing to this directory is \$VSC\_DATA. 
+The environment variable pointing to this directory is \$VSC\_DATA.
 This volume is shared across all
 clusters of the VSC. There are however no guarantees about the
-speed you'll achieve on this volume. For guaranteed fast performance and very heavy I/O, 
+speed you'll achieve on this volume. For guaranteed fast performance and very heavy I/O,
 you should use the scratch space instead.
+\ifgent
+If you are running out of quota on your \$VSC\_DATA filesystem you can
+request a VO (Virtual organisation). VO members get an extra DATA filesystem: \$VSC\_DATA\_VO)
+We can give higher quota values for these filesystems, up to 100TiB is no exception.
```

Comment:
I'm still not sure what we should use. For storage, use TB, not TiB, even though we actually allocate quota in factors of powers of 2 :)